# 3.4

## Exception Handling

Python uses exceptions to handle errors that occur at runtime.
You can use a try/except block to catch exceptions and prevent your program from crashing:

````python
try:
    print(10 / 0)
except ZeroDivisionError:
    print("You can't divide by zero!")
````
